.kokoro/nightly/create-versions-csv.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#input -> text file that lists versionless coordinates
+#
+#example ->
+#com.google.cloud:google-cloud-storage
+#com.google.cloud:google-cloud-vision
```

Comment:
I don't think you need this input because you can get them from Libraries BOM.

Reply:
updated comments.